src/OctoshiftCLI.Tests/ado2gh/Commands/AbortMigration/AbortMigrationCommandTests.cs

```diff
@@ -0,0 +1,20 @@
+using FluentAssertions;
+using OctoshiftCLI.AdoToGithub.Commands.WaitForMigration;
+using Xunit;
+
+namespace OctoshiftCLI.Tests.AdoToGithub.Commands.AbortMigration;
+
+public class AbortMigrationCommandTests
+{
+    [Fact]
+    public void Should_Have_Options()
+    {
+        var command = new AbortMigrationCommand();
+        command.Should().NotBeNull();
+        command.Name.Should().Be("abort-migration");
+        command.Options.Count.Should().Be(1);
```

Comment:
this seems wrong, it should have 3 options:
`--migration-id`, `--github-pat` and `--verbose`

Reply:
[c39b523](https://github.com/github/gh-gei/pull/1183/commits/c39b52331c265328c19cf7d26cfc3fb381299f94)

Reply:
options count also needs to be updated to 3